install/kubernetes/cilium/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+You have successfully installed Cilium {{ title .Chart.Name }}.
```

Comment:
This prints:

```
You have successfully installed Cilium Cilium.

Your release version is 1.7.90.

For any further help, visit https://docs.cilium.io/en/v1.7/gettinghelp
```

Please avoid printing Cilium twice here.

Reply:
Mybad, will fix it.